backup_and_restore/application_backup_and_restore/backing_up_and_restoring/oadp-backing-up-applications-restic-doc.adoc

```diff
@@ -0,0 +1,53 @@
+// Module included in the following assemblies:
+//
+
+:_content-type: PROCEDURE
+[id="oadp-backing-up-applications-restic-doc"]
+= Backing up applications with Restic
+include::_attributes/common-attributes.adoc[]
+:context: backing-up-applications
+
+toc::[]
+
+If your cloud provider does not support snapshots or if your applications are on NFS data volumes, you can create backups by using Restic.
+
+[NOTE]
+====
+link:https://restic.net/[Restic] is installed by the OADP Operator by default.
+====
+
+Restic integration with OADP provides a solution for backing up and restoring almost any type of Kubernetes volume. This integration is an addition to OADP’s capabilities, not a replacement for existing functionality.
+
+You back up Kubernetes resources, internal images, and persistent volumes with Restic by editing the `Backup` custom resource (CR).
+
+You do not need to specify a snapshot location in the `DataProtectionApplication` CR.
+
+[IMPORTANT]
+====
+Restic does not support backing up `hostPath` volumes. For more information, see link:https://{velero-domain}/docs/v{velero-version}/restic/#limitations[additional Restic limitations].
+====
+
+.Prerequisites
+
+* You must install the OpenShift API for Data Protection (OADP) Operator.
+* You must not disable the default Restic installation by setting `spec.configuration.restic.enable` to `false` in the `DataProtectionApplication` CR.
+* The `DataProtectionApplication` CR must be in a `Ready` state.
```

Comment:
Is this considered a Ready State?
status:
    conditions:
    - lastTransitionTime: "2023-07-23T18:17:29Z"
      message: Reconcile complete
      reason: Complete
      status: "True"
      type: Reconciled
   maybe i missing something

Reply:
@amastbau - please can you help me with what this should be? thanks

Reply:
I would suggest in The `DataProtectionApplication` CR must be with the status `Condition: Reconciled`
@anarnold97 
I wonder how this status condition is referred to in other product documentation
This is a comment status for a CR.